renderer/index.js

```diff
@@ -206,6 +206,9 @@ function dispatch (action, ...args) {
     state.video.isPaused = true
     update()
   }
+  if (action === 'videoPlaying') {
+    ipcRenderer.send('playing-video')
+  }
   if (action === 'videoPaused') {
     ipcRenderer.send('paused-video')
```

Comment:
message/action names as constants in a file required by both main and renderer would do the trick